proto/model/tx.proto

```diff
@@ -38,6 +38,8 @@ message MsgCreateModel {
   string user_manual_url = 14 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
   string support_url = 15 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
   string product_url = 16 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
+  string lsf_url = 17 [(gogoproto.moretags) = "validate:\"required_unless=LsfRevision 0,omitempty,url,startsnotwith=http:,max=256\""];
+  int32 lsf_revision = 18 [(gogoproto.moretags) = "validate:\"required_with=LsfUrl,gte=0,lte=65535\""];
```

Comment:
As stated in https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/service_device_management/DistributedComplianceLedger.adoc#1512-lsfrevision

>  When the first version of the Localized String File is published, the value of this field SHOULD be 1.

So, it looks like no need to have `lsf_revision` field in `MsgCreateModel` as it must be set to `1` by DCL, doesn't it?

Reply:
OK.. then how about we remove lsf_revision field for Addition